sample-apps/donet-sample-app/Controllers/AppController.cs

```diff
@@ -80,9 +78,33 @@ public string Default()
         [Route("/outgoing-sampleapp")]
         public string OutgoingSampleApp()
         {
-            var port = 4567;
+            using var activity = tracer.StartActivity("outgoing-sampleapp");
+            activity?.SetTag("language", "dotnet");
+            activity?.SetTag("signal", "trace");
+            string statusCode = "";
+
+            if (Program.cfg.SampleAppPorts.Length == 0) {
+                var res = httpClient.GetAsync("https://aws.amazon.com/").Result;
+                statusCode = res.StatusCode.ToString();
+            }
+            else {
+                foreach (string port in Program.cfg.SampleAppPorts) {
+                    if (!String.IsNullOrEmpty(port)) {
+                        Console.WriteLine(port);
+                        string uri = "http://127.0.0.1:" + port + "/outgoing-sampleapp";
+                        var res = httpClient.GetAsync(uri).Result;
+                        statusCode = res.StatusCode.ToString();
+                    }
+                }
+            }
             
-            return "Application started!";
+            // Request Based Metrics
+            Startup.metricEmitter.emitReturnTimeMetric(MimicLatency());
```

Comment:
nit: Is there a reason why we mock the latency value rather than measure the actual time? Do the other sample apps also do this?

Reply:
I just checked and about half of the current sample apps mimic the latency value while the other half measure the actual time to handle the request.

Reply:
Would it break the integration tests if we added an additional attribute to signal whether or not this value was `actual` or `mock`?

Reply:
I believe adding an additional attribute will break the integration tests.

Reply:
I figured, we would also need to introduce this into the spec first but it would be a welcome change imo. This attribute would help users understand if the latency is real or not. I also wonder if the behavior to simulate latency is defined in the spec. 

Can you create an issue to track this? "Add attribute to to latency metric to track whether or not it is actual or mocked."